packages/cli/src/lib/setup/setupUpload.ts

```diff
@@ -216,23 +255,19 @@ function Upload(options) {
                 }
             }
         }
-    };
+    }
 
     /**
      * Uploads a file
-     *
-     * @param {string} source
-     * @param {string} target
-     * @return {Promise<string>}
      */
-    this.uploadFile = async (source, target) => {
+    async uploadFile(source: string, target: string): Promise<string> {
         target = target.replace(/\\/g, '/');
         source = source.replace(/\\/g, '/');
         if (target[0] === '/') {
             target = target.substring(1);
         }
         if (target[target.length - 1] === '/') {
-            let name = source.split('/').pop();
+            let name = source.split('/').pop() as string;
```

Comment:
```suggestion
            let name = source.split('/').pop()!;
```
You already know it is string or undefined. `!` removes the undefined.